### PR TITLE
FOUNDATIONS: Add Order

### LINKED ISSUE #78: FOUNDATIONS: Add Order

### PR DESCRIPTION
**Pull request by StudentofTATU**
closes #78

### CODE REVIEW

File: NajotBooking.Api/Models/Orders/Exceptions/FailedOrderServiceException.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+using System;
+using Xeptions;
+
+namespace NajotBooking.Api.Models.Orders.Exceptions
+{
+    public class FailedOrderServiceException : Xeption
+    {
+        public FailedOrderServiceException(Exception innerException)
+            : base(message: "Failed profile service occurred, please contact support", innerException)
```
Comment (GulchexrasGithub) [inline NajotBooking.Api/Models/Orders/Exceptions/FailedOrderServiceException.cs:14]: it must be Order not profile

File: NajotBooking.Api/Models/Orders/Exceptions/OrderServiceException.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+using System;
+using Xeptions;
+
+namespace NajotBooking.Api.Models.Orders.Exceptions
+{
+    public class OrderServiceException : Xeption
+    {
+        public OrderServiceException(Exception innerException)
+            : base(message: "Profile service occurred, contact support.", innerException)
```
Comment (GulchexrasGithub) [inline NajotBooking.Api/Models/Orders/Exceptions/OrderServiceException.cs:14]: It's too, change to Order

File: NajotBooking.Api/Services/Foundations/Orders/OrderService.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+{
+    public partial class OrderService : IOrderService
+    {
+        private readonly IStorageBroker storageBroker;
+        private readonly ILoggingBroker loggingBroker;
+
+        public OrderService(IStorageBroker storageBroker, ILoggingBroker loggingBroker)
+        {
+            this.storageBroker = storageBroker;
+            this.loggingBroker = loggingBroker;
```
Comment (GulchexrasGithub) [inline NajotBooking.Api/Services/Foundations/Orders/OrderService.cs:21]: You need DateTime broker

**Review by GulchexrasGithub — APPROVED** [review]

**Review by Nodirkhan — COMMENTED** [review]
LGTM

**Pull request merged**